docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -0,0 +1,202 @@
+hot-restart-persistence-hazelcast-2   Bound    pvc-64314d82-da7a-4e38-bd2d-e770a63dc4e8   20Gi       RWO            standard       1m
+----
+--
+
+== Take a Single Hot Backup
+--
+. Create the HotBackup resource:
++
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/hot-backup.yaml[]
+----
+
+. Apply the resource:
++
+[source,bash]
+----
+kubectl apply -f ./hot-backup.yaml
+----
+
+--
+
+== Take Scheduled Hot Backup
+
+You can also take backups on schedule using the `spec.schedule` field.
+--
+. Create the HotBackup resource with `schedule` field filled:
++
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/hot-backup-scheduled.yaml[]
+----
++
+The field needs to take a valid cron expression. Some examples:
++
+[cols="1,1"]
+|===
+| ```30 10 * * *```
+| On 10:30 AM every day
+
+| ```0, 0, 1,15,25 * *```
+| On 1st, 15th and 25th each month, midnight
+
+| ```@monthly```
+| Once a month, first of the month, midnight
+|===
+
+
+. Apply the resource:
++
+[source,bash]
+----
+kubectl apply -f ./hot-backup-scheduled.yaml
+----
+--
+
+== Restore from HotBackup
+
+. Delete the Hazelcast cluster:
++
+[source,bash]
+----
+kubectl delete hazelcast hazelcast
+----
+
+. Check PVCs are still there:
++
+[source,bash]
+----
+kubectl get pvc -l app.kubernetes.io/instance=hazelcast
+----
++
+[source,bash]
+----
+NAME                                  STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+hot-restart-persistence-hazelcast-0   Bound    pvc-116b4084-a436-4462-b413-511b77df307b   20Gi       RWO            standard       6m
+hot-restart-persistence-hazelcast-1   Bound    pvc-a7711b6b-dcbf-45cb-9577-8ce6b1892f2f   20Gi       RWO            standard       5m
+hot-restart-persistence-hazelcast-2   Bound    pvc-64314d82-da7a-4e38-bd2d-e770a63dc4e8   20Gi       RWO            standard       5m
+
+----
+
+. Reapply the Hazelcast resource:
++
+[source,bash]
+----
+kubectl apply -f ./hazelcast-persistence.yaml
+----
+
+. Same PVCs will be bound to the new pods:
++
+[source,bash]
+----
+kubectl get pods -l app.kubernetes.io/instance=hazelcast -ojsonpath='{.items[*].spec.volumes[?(@.name=="hot-restart-persistence")].persistentVolumeClaim.claimName}'
+
+----
++
+[source,bash]
+----
+hot-restart-persistence-hazelcast-0 hot-restart-persistence-hazelcast-1 hot-restart-persistence-hazelcast-2
+----
+
+=== Restore Policy
+
+Restore policy can be adjusted by setting `clusterDataRecoveryPolicy` field. Supported
+options are:
+
+[cols="1,1"]
+|===
+| ```FullRecoveryOnly```
+| Does not allow partial start of the cluster.
+
+| ```PartialRecoveryMostRecent```
+| Allows partial start with the members that have most recent partition table.
+
+| ```PartialRecoveryMostComplete```
+| Allows partial start with the member that have most complete partition table.
+
+| ```ForceStart```
```

Comment:
In Platform, force-start must be triggered manually. In K8s Operator, how does the `ForceStart` policy work? 

Are you also able to trigger partial-start manually when you configure one of the `Partial` policies?

Reply:
The operator detects member crash during the restore startup and do the triggering automatically if `autoStart` is enabled in API. 
I will clarify it with @SeriyBg and @hasancelik (PR is in review) and update it accordingly.

Reply:
@hakanmemisoglu this option won't be supported for `recovery-policy` and will be removed in the ForceStart PR, as we have a separate filed for auto force start